### PR TITLE
Revert "add install_requires to setup.py and update url"

### PR DESCRIPTION
**Pull request by jonwright**
Reverts FABLE-3DXRD/xfab#20 : I am getting numpy 1.21.5 on a new windows conda

### CODE REVIEW

**Comment by AxelHenningsson** [comment]
Sorry if I messed up anything! :confused: Was thinking that we would like to have an `install_requires `for the pypi distributions to work, so that we eventually can have the latest version of the package pip installable so that we eventually can have a CI that runs pytests.

Perhaps we want to go a bit more loosely with the versions in the `install_requires `? If not I guess just reverting this will resolve any issues?

**Comment by AxelHenningsson** [comment]
What about being as loose as dropping all versions:

```
  install_requires = ['numpy',
                      'six',
                      'pycifrw']
```

**Comment by jonwright** [comment]
Sure - thanks for working on this! The generic install_requires seems better for setup.py. For reproducible results using xfab it would make sense to list all packages in a conda or venv. I think h5py (for example) is more fussy about the numpy versions. For pycifrw I have no idea if something changed recently?

**Comment by AxelHenningsson** [comment]
Does this feel alright? Will try and make the GitHub Workflow succeed on Ubuntu latest next thing.

Cheers 
Axel

File: setup.py [hunk 1]
```diff
@@ -25,4 +25,7 @@
            "scripts/tweakdetpars.py",
            "scripts/flt_remove_beam.py",
            "scripts/flt_split_phases.py"]
```
Comment (jonwright) [inline setup.py:27]: comma needed at the end of line 27 ?

**Comment by jonwright** [comment]
To fix the CI I think it is better to get a binary wheel for numpy. Probably going python 3.9 is better as I think most of the community has not gone over 3.10?

**Comment by jonwright** [comment]
So I will merge like this and leave you to debug the CI - it makes sense to bump version of xfab to 0.0.5 and add yourself as a maintainer too ... I can try to find the keys to pypi if you want to do a release when things look good?

**Pull request merged**